### ISSUE TITLE
card_light and card_cover not loading on mobile

### ISSUE THREAD

**Issue by jaku2019** [UI-Lovelace-Minimalist/UI#1097]
Hello,
I'm struggling with an annoying problem from 2 months, so I decided I would ask here for its solution.

**Descripion**
"card_light" and "card_cover" not loading on mobile. Sometimes only a part of them is showed.

**To Reproduce**

1. Set card_light in the yaml
2. Open Home Assistant app
3. card_light and card_cover won't be visible in the app **[1]** (in opose to webbrowser **[2]**, where everything is displayed as it should)
4. See the error.

**Expected behavior**
Light cards (aswell as rolling shutters cards) should appear in the mobile app.
**Screenshots**
![1](https://user-images.githubusercontent.com/55255300/202930510-a2d3e32c-5182-4aee-a4bc-c89a7790e7c4.PNG)
![2](https://user-images.githubusercontent.com/55255300/202930525-81fd3894-e31f-4d6b-bdb9-714f37518eb0.PNG)
![3](https://user-images.githubusercontent.com/55255300/202931224-99001a63-9ee1-4dbd-af13-0ddb6d3b8271.gif)

**Additional context**
I want to add that after installing "Minimalist UI" from HACS, I had to install "button-card" - otherwise it didn't want to work.

My OS is Android 10. After primal install of Home Assistant everything is loaded correctly, but after restart of the app, all card_light buttons dissapear. Reinstallation of the app solves the problem temporarily (restarting the app causes the same problem as described above).

I'm also having troubles with the filling animation in card_light and card_cover. When I press the middle of any button, I see a weird fill animation that fills only 3/4 of the button, whereas when I touch the edge of any button, the rectangle that encircles the card is showed **[3]**. It's visible only on mobile after the first installation of the app, when those cards are still displayed.
Additionaly I see that the Minimalist UI is loading pretty long on my phone - after the start it loads a row after row until everything is displayed.

Regards,
Jakub

**Comment by basbruss** [comment]
Hi @jaku2019 

Have you made sure you have installed all other dependencies except button card? 
Issues on mobile can be cache related and on android, it's best to remove all cached files from your phone's settings menu:
settings > apps > homeassistant


> I'm also having troubles with the filling animation in card_light and card_cover

The filling automation is not a bug. It's a result of enabling different click actions on certain card parts (e.q. for the name, icon, and outer edge). You can find more information on that when looking for custom_actions

**Comment by jaku2019** [comment]
>Hi @jaku2019

Thank you very much for reply, @basbruss !
> Have you made sure you have installed all other dependencies except button card? Issues on mobile can be cache related and on android, it's best to remove all cached files from your phone's settings menu: settings > apps > homeassistant

I've installed browser_mod and button-card. [According to the documentation](https://ui-lovelace-minimalist.github.io/UI/setup/download/#hacs-download), no other dependencies are needed.

When it comes to cache on Android: I've already tried that. After removing app's cache, the app acts like it was just installed. After restarting the problem occurs again. Basically, I've tried HassIO with Minimalist UI on other smartphones (with the newest Android) and the problem was the same there, so I think it's host-related.

> The filling automation is not a bug. It's a result of enabling different click actions on certain card parts (e.q. for the name, icon, and outer edge). You can find more information on that when looking for custom_actions

I was aware about variousity of click actions, but I didn't know that they have different animations assigned. Even after disabling some of the click actions in appropiate file, those weird animations show up. Are you shure that the one on which a rectangle that surrounds the whole card is _a feature not a bug_ :) ?

![freeze-frame](https://user-images.githubusercontent.com/55255300/204161502-6f6973b9-59c5-499f-87a5-57808dbfaebb.png)
